src/webview/css/css.js

```diff
@@ -108,6 +108,16 @@ hr {
 .message-brief {
   padding: 0 1em 1em 4em;
 }
+.message-brief > .content > :first-child {
+  clear: right;
+}
+.message-brief > .content > :last-child::after {
+  content: var(--time);
```

Comment:
Have you confirmed that CSS variables are supported on all platforms that we support?

Reply:
I had not, thanks for reminding! I have since confirmed that it works from the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties).